### PR TITLE
More comprehensive codebuild generation

### PR DESCRIPTION
**Pull request by markrawls**

### CODE REVIEW

File: zygoat/components/resources/testing.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+phases:
+  install:
+    runtime-versions:
+      python: 3.8
+  pre_build:
+    commands:
+      - scripts/ci-start.sh
+  build:
+    commands:
+      - docker-compose exec -T backend python manage.py test
```
Comment (Ian-MacLeod) [inline zygoat/components/resources/testing.yml:12]: Do we not need a wait_for_db command here?

Comment (markrawls) [reply zygoat/components/resources/testing.yml]: Added

File: zygoat/components/codebuild.py [hunk 2]
```diff
@@ -1,12 +1,25 @@
...
 
 
-class CodeBuildDefault(CodeBuild):
-    filename = "codebuild.yml"
+class Testing(CodeBuildFile):
+    filename = "testing.yml"
 
 
-codebuild = CodeBuildDefault()
+codebuild = CodeBuild(sub_components=[Security(), Linting(), Testing()])
```
Comment (ambikads) [inline zygoat/components/codebuild.py:25]: Should we split testing into frontend-testing and backend-testing?

Comment (markrawls) [reply zygoat/components/codebuild.py:25]: Right now we currently don't ship with a frontend testing command, otherwise I would split it into those.

Comment (ambikads) [reply zygoat/components/codebuild.py:25]: Understand, My comment was assuming the frontend testing will eventually come and we don't have to rename files when that happens.

Comment (markrawls) [reply zygoat/components/codebuild.py]: Updated

**Comment by ambikads** [comment]
lgtm as well

**Pull request merged**